frontend/src/views/instructors/editable-instructors-list.js

```diff
@@ -42,6 +55,21 @@ function EditableInstructorsList(props) {
         );
     }
 
+    function DeleteButtonCell({ original: instructorInfo }) {
+        const { id } = instructorInfo;
+        const disabled = workingInstructors.has(id);
+
+        function onClick() {
+            deleteInstructor({ id });
+        }
+
+        return (
+            <Button variant="danger" onClick={onClick} disabled={disabled}>
```

Comment:
I don't know what it looks like, but I'm guessing it stands out? Make the delete button appear muted until you hover. Also, all buttons should have a `title`

Reply:
Disabled greys out the icon:
![image](https://user-images.githubusercontent.com/30083877/86407871-dcd67000-bc83-11ea-9cbe-ac3ab3484d8f.png)

Reply:
That's still too much. Just have a grey "X" until you hover and then have it turn red. We will very rarely be deleting instructors.

Reply:
OK. Do you still want a column for it? It looks weird with a big column with just a tiny "x" inside.